crates/rerun_c/src/lib.rs

```diff
@@ -52,6 +52,8 @@ pub struct CStoreInfo {
     pub application_id: *const c_char,
 
     pub store_kind: CStoreKind,
+
+    pub default_enabled: bool,
```

Comment:
This doesn't belong here, it breaks the fact that `CStoreInfo` is a mirror of `StoreInfo`... Do you really need to store this here?

Reply:
Got it -- we weren't exactly using CStoreInfo as a mirror of StoreInfo in any meaningful way given we just deconstruct it to pass the arguments into the builder. I had just interpreted it as something more like "RecordingStreamConstructionArgs". But I'll just move it to a function argument instead.